source/Popup/index.tsx

```diff
@@ -183,7 +181,13 @@ function downloadZestScript(zestScriptJSON: string, title: string): void {
   closePopup();
 }
 
-function handleSaveScript(): void {
+async function handleSaveScript(): Promise<void> {
```

Comment:
Better fix this in its own PR, it's unrelated to the changes being added here.

Reply:
okay